src/main/java/sirius/web/http/WebServer.java

```diff
@@ -829,6 +829,16 @@ public void gather(MetricsCollector collector) {
                          queueTime.getAndClear(),
                          "ms");
         collector.metric("http_websockets", "http-websockets", "Open Websockets", websockets.get(), null);
+        collector.metric("pooled_byte_buffer_used_heap_mem",
+                         "pooled-byte-buffer-used-heap-mem",
+                         "Pooled byte buffer allocation used heap memory",
+                         PooledByteBufAllocator.DEFAULT.metric().usedHeapMemory() / 1024d / 1024d,
```

Comment:
1024 -> MiB, 1000 -> MB … please divide by 1000 as we usually do, following SI units

```suggestion
                         PooledByteBufAllocator.DEFAULT.metric().usedHeapMemory() / 1000d / 1000d,
```

Reply:
That will give unclear values, compared to how one passes settings to the JVM, such as -XX:MaxDirectMemory=16g
If so, happy to change the unit from MB to MiB. What others say?

Reply:
Other metrics are also calculated this way. See https://github.com/scireum/sirius-db/blob/03a1f8ab9cf2ab91e839753bae3b66f0e925cad0/src/main/java/sirius/db/redis/RedisMetricProvider.java

Reply:
Well, that is not good 😅 ... I don't mind to write MiB, but the prefixes need to match the factors, and we should fix it everywhere. 🙃

Reply:
Go tell Microsoft that Windows calculates files wrong, since, EVER! 🤣

Reply:
Will track in a follow up ticket.

Reply:
[SIRI-625](https://scireum.myjetbrains.com/youtrack/issue/SIRI-625)